### ISSUE TITLE
implicitmessage task consistently fails

### ISSUE THREAD

**Issue by iand** [filecoin-project/sentinel-visor#644]
### Describe the bug:

Running a walk that includes `implicitmessage` task I get the following error on every tipset visited:

2021-08-13T13:34:46.562Z        ERROR   visor/chain     chain/indexer.go:319    failed to extract messages      {"height": 1016876, "error": "failed to extract message execution for tipset [bafy2bzacebg52f4ozfkaiar4bklliy6i6qnfcqhxoj62xjd6aebfzbnqgljyw bafy2bzacecgnmsvaj6aeubbeg55feys3mighhkc6zlnkakxpeyxbracueeozq]: failed to find execution trace", "errorVerbose": "failed to extract message execution for tipset [bafy2bzacebg52f4ozfkaiar4bklliy6i6qnfcqhxoj62xjd6aebfzbnqgljyw bafy2bzacecgnmsvaj6aeubbeg55feys3mighhkc6zlnkakxpeyxbracueeozq]:\n    github.com/filecoin-project/sentinel-visor/lens/lily.(*LilyNodeAPI).GetMessageExecutionsForTipSet\n        /home/ubuntu/sentinel-visor/lens/lily/impl.go:195\n  - failed to find execution trace:\n    github.com/filecoin-project/sentinel-visor/lens/lily/modules.init\n        /home/ubuntu/sentinel-visor/lens/lily/modules/statemanager.go:21"}

### Steps to Reproduce:

Start a walk from 1016870 to 1016879 with tasks=implicitmessage (with csv storage but this error happens before extraction or persistence)

### Visor Version: master (8e6f33328e7c8ee3c1ec308d2e0b351435ea6bb9)

As an aside: why is it `implicitmessage` while the `messages` task  is pluralised (as are others such as `actorstatesraw`)?

**Comment by iand** [comment]
No error details are recorded in `visor_processing_reports` so I can't diagnose further:

csv output:
```
1016876,bafy2bzacecwcfmms5z23kfxbrl7osgf4422w3daly2q25dmbywdumkvfai2rc,export-1014000-1016879,implicitmessage,2021-08-13T13:32:57.902Z,2021-08-13T13:34:46.562Z,ERROR,,{}
```

**Comment by frrist** [comment]
https://github.com/filecoin-project/sentinel-visor/pull/639

**Comment by frrist** [comment]
The messages processors all fail to write errors for some reason https://github.com/filecoin-project/sentinel-visor/issues/597

**Comment by frrist** [comment]
Closed via #639
